harness/harness.go

```diff
@@ -104,7 +117,14 @@ func (c *H) flushToParent(format string, args ...interface{}) {
 
 	c.mu.Lock()
 	defer c.mu.Unlock()
-	io.Copy(p.w, &c.output)
+	b := c.output.Bytes()
+	io.Copy(p.w, bytes.NewBuffer(b))
+}
+
+func (c *H) publishToReporters(name, status string) {
+	if !c.hasSub {
```

Comment:
Why skip tests-that-have-subtests? That is what this is doing, right?

Reply:
All of the tests are at the same level, this makes it so that the test groupings don't get reported as individual tests (i.e. `coreos.flannel.vxlan` & `coreos.flannel.udp` are reported but `coreos.flannel` is not)